src/views/FarmAuction/hooks/useCongratulateAuctionWinner.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState, useEffect } from 'react'
+import { useWeb3React } from '@web3-react/core'
+import { Auction, AuctionStatus, Bidder } from 'config/constants/types'
+import { useFarmAuctionContract } from 'hooks/useContract'
+import { processAuctionData, sortAuctionBidders } from '../helpers'
+
+interface WonAuction {
+  auction: Auction
+  bidderData: Bidder
+}
+
+const useCongratulateAuctionWinner = (currentAuction: Auction, bidders: Bidder[]): WonAuction => {
```

Comment:
This hook checks if user won either:
current auction if it is closed
or previous auction
Used to display CongratulationsCard